databases/week2/cocoa_productionTABLE.sql

```diff
@@ -0,0 +1,40 @@
+
+CREATE TABLE cocoa_production (
+	`product_id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+	`product_name` VARCHAR(255) NOT NULL,
+	`start_date` DATETIME NOT NULL,
+	`due_date` DATETIME NULL DEFAULT NULL,
+	`product_location` FLOAT NOT NULL,
+    `price` DECIMAL(10,2) NOT NULL CHECK (price >= 0),
+     PRIMARY KEY (`product_id`,`product_name`),
+     FOREIGN KEY (`price`) REFERENCES customer(`order_id`)
+)ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
```

Comment:
This statement gives me an error: `Error Code: 1068. Multiple primary key defined` See if you can figure out the problem